angular.json

```diff
@@ -80,7 +80,13 @@
                 }
               ],
               "serviceWorker": false,
-              "ngswConfigPath": "ngsw-config.json"
+              "ngswConfigPath": "ngsw-config.json",
+              "fileReplacements": [
+                {
+                  "replace": "src/app/core/store/store-devtools.module.ts",
+                  "with": "src/app/core/store/store-devtools.module.prod.ts"
```

Comment:
Can you rename the file to `.production.ts` so it has the same name as the configuration and can be replaced automatically later on?

Reply:
Yes this can be done and might be more consistent.